assets/model_monitoring/amlcompute_components/feature_attribution_drift/feature_importance_metrics_command/spec.yaml

```diff
@@ -1,20 +1,18 @@
 $schema: http://azureml/sdk-2-0/SparkComponent.json
-type: spark
+type: command
 
 name: feature_importance_metrics_command
-version: 0.0.1
 display_name: Feature importance
-is_deterministic: true
 description: Feature importance for model monitoring.
+version: 0.0.2
+is_deterministic: true
 
 code: ../../src
-entry:
-  file: ./feature_importance_metrics/compute_feature_importance.py
+environment: azureml:model_monitor_spark_standalone_feature_attribution:1
```

Comment:
TODO: Will update this again once the environments are available in azureml registry.